mongod.conf

```diff
@@ -0,0 +1,11 @@
+storage:
+  dbPath: /data/db
+
+systemLog:
+  destination: file
+  path: /data/configdb/log/mongod.log
```

Comment:
A little bit worried there, we may have huge mongod log, could we investigate whether the log will be rotate and limit the size by itself?